controllers/review.controller.js

```diff
@@ -0,0 +1,12 @@
+const { seeReviewsById } = require("../models/review.model");
+
+exports.viewReviewsById = (req, res, next) => {
+  const { review_id } = req.params;
+  seeReviewsById(review_id)
+    .then((review) => res.status(200).send({ review }))
+    .catch(next);
+};
+
+exports.notFoundErr = (req, res, next) => {
```

Comment:
This error isn't specifically tied to reviews, so it should be moved into app.js, or ideally its own error-handling file.